tests/integration/distribution.go

```diff
@@ -242,8 +242,8 @@ func (s *CCVTestSuite) TestEndBlockRD() {
 	}
 }
 
-// getEscrowBalance gets the current balances in the escrow account holding the transferred tokens to the provider
-func (s CCVTestSuite) getEscrowBalance() sdk.Coins {
+// getEscrowBalance gets the current balances in the escrow account holding the transfered tokens to the provider
```

Comment:
Are the changes here and onward intentional? If so could we add to the description of the PR why they are needed to bump sdk

Reply:
that's intentional.  It set off the copy locks linter.  So I suppressed it because we are copying locks in the test.

I don't think there's any additional notation needed, but sure